tests/auth_oidc_test.go

```diff
@@ -199,6 +199,24 @@ func Test_Authenticate_OIDC_Auth(t *testing.T) {
 		ctx.Req.NotEmpty(outTokens.IDTokenClaims)
 		ctx.Req.NotEmpty(outTokens.AccessToken)
 		ctx.Req.NotEmpty(outTokens.RefreshToken)
+
+		t.Run("access token has expected values", func(t *testing.T) {
```

Comment:
all these tests have the same description. it **might** be nice to differentiate em?

Reply:
They are sub-runs so they show in the context of their parent test, both on the CLI and in IDE test runs.